src/citrine/resources/process_template.py

```diff
@@ -89,7 +90,7 @@ def __init__(self,
                                        allowed_names=allowed_names)
 
     @classmethod
-    def _build_child_objects(cls, data: dict, session: Session = None):
+    def _build_child_objects(cls, data: dict, data_with_soft_links, session: Session = None):
```

Comment:
Surprised this was working before without this argument. Do you know what happened?

Reply:
The actual implementations of `_build_child_objects` don't use `data_with_soft_links` or `session`. The data with soft links was therefore getting passed as `session`, which went unnoticed because Python doesn't do type checking and the argument wasn't used.